client/mass/about.ts

```diff
@@ -131,11 +132,17 @@ class MassAbout {
 									})
 								}
 
-							app.dispatch({
+							await app.dispatch({
 								type: 'app_refresh',
 								subactions
 							})
-						} else app.dispatch({ type: 'cohort_set', activeCohort: i })
+						} else await app.dispatch({ type: 'cohort_set', activeCohort: i })
+
+						nodes.forEach((n, j) => {
+							select(n)
+								.select('input')
+								.property('checked', j === state().activeCohort)
```

Comment:
works now. state() looks weird and is defined up as s in line 112. I would rather use this.app.getState() than using this shortcut.

Reply:
Actually, better not to await as these actions may take a while, and you know the cohort beforehand, it is i

Reply:
I thought a bit more about this logic, the node is being checked in this action, the problem is this line that runs on main, just removing the line fixes the issue
`		//this.dom.cohortInputs.property('checked', (d, i) => i === this.activeCohort)
`